src/http/resource.ts

```diff
@@ -1,69 +1,47 @@
 import { Drash } from "../../mod.ts";
 
 /**
- * @memberof Drash.Http
- * @class Resource
- *
- * @description
- *     This is the base resource class for all resources. All resource classes
- *     must be derived from this class.
+ * This is the base resource class for all resources. All resource classes
+ * must be derived from this class.
  */
 export class Resource implements Drash.Interfaces.Resource {
   /**
-   * @description
-   *     A property to hold the middleware this resource uses.
+   * A property to hold the middleware this resource uses.
    *
-   *     All derived middleware classes MUST define this property as static
-   *     (e.g., static middleware = ["MiddlewareClass"];)
-   *
-   * @property string[] middleware
+   * All derived middleware classes MUST define this property as static
+   * (e.g., static middleware = ["MiddlewareClass"];)
    */
   public middleware: { after_request?: []; before_request?: [] } = {};
```

Comment:
making an issue to remove this logic completely. we're not using a static property on resources anymore to define resource-level middleware.

Reply:
I already removed them from the docs. Will handle the issue next.